src/device-monitoring/config/db_connection.py

```diff
@@ -13,5 +13,5 @@
 
 def connect_mongo(tenant):
     client = MongoClient(app_configuration.MONGO_URI)
-    db = client[f"{app_configuration.DB_NAME}_{tenant.lower()}"]
+    db = client[f"{app_configuration.DB_NAME}"]
```

Comment:
Hi @BenjaminSsempala , we might need to review this since this underscore implementation is a common thing that has existed for many years.

You might just want to update the respective ENVS...by removing the` _{TENANT}` accordingly.

Reply:
Alright No worries @Baalmart 
Let me update the envs instead

Reply:
Although, it might be confusing in the future since the variable in the .env is named `DB_NAME_STAGE=device_mon_stage` but the actual db name is `device_mon_stage_airqo` but there's no way of knowing that until after you've inspected the code
@Baalmart